src/components/v5/common/Table/partials/BurgerMenu.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from 'react';
```

Comment:
We have task to create reusable BurgerMenu, I will create this so we will have to replace this.

Reply:
After we'll merge you PR I'll replace it to your component